tests/serializers/tests.py

```diff
@@ -280,7 +280,14 @@ def test_serialize_prefetch_related_m2m(self):
         with self.assertNumQueries(4):
             serializers.serialize(
                 self.serializer_name,
-                Article.objects.prefetch_related("categories", "meta_data", "topics"),
+                Article.objects.prefetch_related(
+                    "meta_data",
+                    "topics",
+                    Prefetch(
+                        "categories",
+                        queryset=Category.objects.prefetch_related("meta_data"),
```

Comment:
I realize that this prefetch isn't realistic given this property on the `Category` model is not a many-to-many relationship (in this context `select_related` is more appropriate), but I was trying to avoid introducing any new models or properties to the sample data.

If folks would prefer this to be more realistic, I had considered potentially modifying `meta_data` to be a many-to-many relationship with `Category` since multiple pieces of meta data existing on a category is something that's also a common pattern.

Open to other suggestions though 😄